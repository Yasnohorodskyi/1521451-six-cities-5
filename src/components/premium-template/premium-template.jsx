import React from 'react';
import PropTypes from 'prop-types';

const PremiumTemplate = ({className}) => {
  return (
    <div className={className}>
      <span>Premium</span>
    </div>
  );
};

PremiumTemplate.propTypes = {
  className: PropTypes.string.isRequired,
};

export {PremiumTemplate};
export default PremiumTemplate;
