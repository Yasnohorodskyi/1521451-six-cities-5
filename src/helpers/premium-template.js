import React from 'react';
import PropTypes from 'prop-types';

const premiumTemplate = (className) => {
  return (
    <div className={className}>
      <span>Premium</span>
    </div>
  );
};

premiumTemplate.propTypes = {
  className: PropTypes.string.isRequired,
};

export {premiumTemplate};
