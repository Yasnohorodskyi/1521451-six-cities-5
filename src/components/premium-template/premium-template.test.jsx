import React from "react";
import renderer from "react-test-renderer";

import {PremiumTemplate} from "./premium-template";


describe(`PremiumTemplate tests`, () => {
  it(`Render PremiumTemplate`, () => {
    const tree = renderer
      .create(
          <PremiumTemplate className={{}}/>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
