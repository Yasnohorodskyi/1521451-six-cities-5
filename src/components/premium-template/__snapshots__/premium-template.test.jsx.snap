// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PremiumTemplate tests Render PremiumTemplate 1`] = `
<div
  className={Object {}}
>
  <span>
    Premium
  </span>
</div>
`;
